examples/gno.land/r/x/grc20_dynamic_call/registry/registry_test.gno

```diff
@@ -0,0 +1,163 @@
+package registry
+
+import (
+	"std"
+	"testing"
+
+	"gno.land/r/x/grc20_dynamic_call/bar"
+	"gno.land/r/x/grc20_dynamic_call/baz"
+	"gno.land/r/x/grc20_dynamic_call/foo"
+
+	"gno.land/r/demo/users"
+
+	_ "gno.land/r/x/grc20_dynamic_call/wrapper"
```

Comment:
I wonder why? 🤔  

I suggest not importing `wrapper` in `registry`. Since ideally you will be publishing `wrapper`(s) after `registry`? also, this might lead to import cycle.  
```suggestion
```